packages/xo-web/src/common/xo/migrate-vm-modal/index.js

```diff
@@ -213,15 +378,20 @@ export default class MigrateVmModalBody extends BaseComponent {
     this.setState({ migrationNetworkId: migrationNetwork.id })
 
   render () {
-    const { vdis, vifs, networks } = this.props
+    const { vmVdis, vmVifs, networks, snapshots } = this.props
     const {
       doNotMigrateVdis,
       host,
       intraPool,
+      mapVdisSrs,
       mapVifsNetworks,
       migrationNetworkId,
       targetSrs,
     } = this.state
+
+    const displayVdis = host !== undefined && !doNotMigrateVdis
+    const displayVifs = intraPool === false
```

Comment:
It's not clear to me why and when `intraPool` should be `undefined`. Is it only when `host` is `undefined`? If that's the case, then change it to `host !== undefined`.

Reply:
`intraPool` can be `undefined` when no `host` selected and can be `false` when  `vm.$pool !== host.$pool`